app/konnect/deployment/import.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Import Kong Gateway Entities into Konnect Cloud
+no_version: true
+---
+
+You can attach any edition of self-managed {{site.base_gateway}} to
+{{site.konnect_saas}} as a data plane, and use Konnect as your control plane.
+
+Use [decK](/deck/) to import Kong Gateway entity configuration into a runtime
+group in your {{site.konnect_short_name}} organization.
+
+Afterward, you must manually move over:
+* Dev Portal files, developer accounts, and applications
+* Convert RBAC roles and permissions into {{site.konnect_short_name}} teams
+* Certificates
+* Custom plugins
+
+You cannot import [unsupported plugins](/konnect/configure/servicehub/manage-plugins/#plugin-limitations).
+
+## Prerequisites
+* {{site.konnect_saas}} [account credentials](/konnect/getting-started/access-account/).
+* [**Organization Admin**](/konnect/org-management/teams-and-roles) permissions.
+* decK v1.12 or later [installed](/deck/latest/installation/).
+
+## Import entity configuration
+
+Use deck to import entity configurations into a runtime group.
+
+1. Make sure that decK can connect to your {{site.konnect_short_name}} account:
```

Comment:
Thought: Should we also show what a successful ping looks like?

I feel like establishing the initial connection is a big step to overcome in using the CLI.

Reply:
Nvm, this is in the other guide. Pls ignore.

Reply:
No, you're probably right. Why not have it in both? I'll add.